Azure/sap-hana
# Add test case in pipeline for anydb

## Problem
We current do not have test for anydb.

## Solution
1. Introduce feature branch - all feature branch should start with `feature/` and followed with feature name (updated in contributing guideline).
1. Add PR trigger for all feature branch.
1. Add testcases for anydb (single node and HA), but only run if PR or build is against feature branch `feature/anydb`.

## Test
1. Setup pipeline against my own branch, see anydb test cases skipped:
```
createAllNew_AnyDB_SN_SLES12


Started: Today at 12:19 PM
Duration: 5m 27s
Evaluating: eq(variables['branchName'], 'feature/anydb')
Expanded: eq('nancyc-anydb-testcase', 'feature/anydb')
Result: False
```

```
createAllNew_AnyDB_HA_SLES12


Started: Today at 12:19 PM
Duration: 14m 27s
Evaluating: eq(variables['branchName'], 'feature/anydb')
Expanded: eq('nancyc-anydb-testcase', 'feature/anydb')
Result: False
```
1. Remove condition and see anydb test cases ran fine - to verify that the template JSON.
1. Further test (eg. anydb PR trigger) can't be done at this stage, will monitor the pipeline once it's merged.
